modAionImpl/src/org/aion/zero/impl/cli/Arguments.java

```diff
@@ -167,6 +167,28 @@
                     "drops all databases except for block and index when not given a parameter or starting from 0 and redoes import of all known main chain blocks")
     private String redoImport = null;
 
+    // data query
+    @Option(
+        names = {"qb", "-q block", "--query block"},
```

Comment:
Parameters with spaces are problematic for the CLI library. We keep the old ones for legacy reasons, but I would recommend not introducing more.

I suggest changing the options to `names = {"qb", "--query-block"}`.
Similarly below: `names = {"qt", "--query-tx"}` and `names = {"qa", "--query-account"}`.